libs/api/domains/cms/src/lib/models/articleReference.ts

```diff
@@ -0,0 +1,29 @@
+import { Field, ObjectType, ID } from '@nestjs/graphql'
```

Comment:
I had to create this file. If we refactor `subArticles` out of the `article` model then we can get rid of this. Webpack failed to resolve the circularity introduced by attaching a parent `article` to the `subArticle` model.
This acted as a patch or "plaster" for that.

Reply:
I see, maybe add a comment to both models to indicate this.
We should also mark the other model as depreciated (e.g. via comment) and set a date for deletion

Reply:
I have added an explanation. The article reference will be removed once subArticles have been successfully removed from articles.